eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -19,7 +20,16 @@ stages:
           Artifacts: ${{ parameters.Artifacts }}
           CtestRegex: ${{ parameters.CtestRegex }}
 
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
```

Comment:
Should we also condition this on LiveTestCtestRegex?  If it isn't set we should do this step correct?